src/index.html

```diff
@@ -7,8 +7,59 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Kaushan+Script&family=Montserrat:wght@400;700&family=Roboto:ital,wght@0,400;0,500;1,400&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#logo">
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="Moyo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">APPLE</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link" >SAMSUNG</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">SMARTPHONES</a>
```

Comment:
```suggestion
            <a href="#apple" class="nav__link is-active">Apple</a>
          </li>

          <li class="nav__item">
            <a href="#samsung" class="nav__link" >Samsung</a>
          </li>

          <li class="nav__item">
            <a href="#smartphones" class="nav__link">Smartphones</a>
```
just use css property text-transform: upper-case